Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -275,17 +276,21 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
     }
 
     private func showLibrary(with homepanelSection: Route.HomepanelSection) {
-        let navigationController = DismissableNavigationViewController()
-        navigationController.modalPresentationStyle = .formSheet
-        let libraryRouter = DefaultRouter(navigationController: navigationController)
+        if let libraryCoordinator = childCoordinators[LibraryCoordinator.self] {
```

Comment:
We can probably add some tests to ensure the child coordinator is being kept alive as we expect it to.

Reply:
Added one test to check if only one coordinator instance is used to present the library vc